File: solution/11xxx/11050 - 이항 계수 1/11050.cpp
#include <bits/stdc++.h>
using namespace std;

int factorial(int num) {
    int rtn = 1;
    for(int i = num; i >= 2; i--) rtn *= i;
    return rtn;
}

int main() {
    int n, k;
    scanf("%d %d", &n, &k);
    printf("%d", factorial(n)/(factorial(k)*factorial(n-k)));
}

File: solution/11xxx/11050 - 이항 계수 1/11050.py
import sys
input = sys.stdin.readline

def factorial(n):
    rtn = 1
    for i in range(1, n+1):
        rtn *= i
    return rtn

n, k = map(int, input().split())
print(factorial(n) // (factorial(k) * factorial(n-k)))